borg_import/rsnapshots.py

```diff
@@ -1,6 +1,20 @@
 import os
+import subprocess
 
 
 def get_snapshots(rsnapshot_root):
-    snapshots = os.listdir(rsnapshot_root)
-    return snapshots
+    output = []
+    for dirname, dirnames, filenames in os.walk(rsnapshot_root):
```

Comment:
that is a full recursion, guess it will yield ALL the dirs/files, including the ones inside your backups.